examples/cluster-py/__main__.py

```diff
@@ -13,34 +13,51 @@
 # TODO specify tags: { "Name": f"{project_name}-2" }
 vpc = Vpc(f"{project_name}-2")
 
-cluster2 = eks.Cluster('eks-cluster',
-                          vpc_id=vpc.vpc_id,
-                          public_subnet_ids=vpc.public_subnet_ids,
-                          public_access_cidrs=['0.0.0.0/0'],
-                          desired_capacity=2,
-                          min_size=2,
-                          max_size=2,
-                          instance_type='t3.micro',
-                          # set storage class.
-                          storage_classes={"gp2": eks.StorageClassArgs(
-                              type='gp2', allow_volume_expansion=True, default=True, encrypted=True,)},
-                          enabled_cluster_log_types=[
-                              "api",
-                              "audit",
-                              "authenticator",
-                          ],)
-
-cluster3 = eks.Cluster(f"{project_name}-3",
-                            vpc_id=vpc.vpc_id,
-                            public_subnet_ids=vpc.public_subnet_ids,
-                            node_group_options=eks.ClusterNodeGroupOptionsArgs(
-                                desired_capacity=1,
-                                min_size=1,
-                                max_size=1,
-                                instance_type="t3.small"
-                            )
+cluster2 = eks.Cluster(
+    "eks-cluster",
```

Comment:
Not sure what's up the resource naming here, but should this be `f"{project_name}-2"` to match the others?

I'm assuming it's to prevent conflicts, but wouldn't this be auto-named? 🤷